lib/spack/docs/packaging_guide.rst

```diff
@@ -2695,6 +2772,33 @@ the build will fail and the install prefix will be removed.  If they
 succeed, Spack considers the build successful and keeps the prefix in
 place.
 
+^^^^^^^^^^^^^^^^
+Build-time tests
+^^^^^^^^^^^^^^^^
+
+Sometimes packages finish to build "correctly" and issues with their run-time
+behavior are discovered only at a later stage, maybe after a full software stack
+relying on them has already been built. To avoid situation of that kind it's possible
+to code build-time tests that will be executed only if the option ``--run-tests``
```

Comment:
code-> run? (somebody who contribute a package usually does not code them in C++ or alike)